discord/gateway.py

```diff
@@ -153,7 +153,7 @@ def run(self) -> None:
             if self._last_recv + self.heartbeat_timeout < time.perf_counter():
                 _log.warning("Shard ID %s has stopped responding to the gateway. Closing and restarting.", self.shard_id)
                 coro = self.ws.close(4000)
-                f = asyncio.run_coroutine_threadsafe(coro, loop=self.ws.loop)
+                f = asyncio.run_coroutine_threadsafe(coro, self.ws.loop)
```

Comment:
What's the point of this change?

Reply:
This is when I removed the `loop` attribute and then re-added it